lib/streamlit/elements/write.py

```diff
@@ -155,8 +158,15 @@ def write(self, *args: Any, **kwargs: Any) -> None:
             height: 300px
 
         """
+        if kwargs:
+            LOGGER.warning(
```

Comment:
I was thinking about using [`warnings`](https://docs.python.org/3/library/warnings.html), but I didn't see any use of this module in the app, so I used `logging.warning` for consistency.
https://github.com/streamlit/streamlit/blob/c0883da8bc7632148c60951f51864e6b8f75e402/lib/streamlit/config_option.py#L268-L282

Here is example of usage `warnings` module in Apache Airflow project:
https://github.com/apache/airflow/blob/8319d70be9ea1e89f3ed57b36840382c84ff407b/airflow/configuration.py#L1470-L1479

Reply:
Yep, I think for now we are not using `warnings` module for everything. Probably something for the tech debt DB to explore the advantages of replacing our `LOGGER.warnings` with warnings module.